.github/workflows/linters.yml

```
@@ -0,0 +1,49 @@
+name: Linters
+
+on:
+  pull_request:
+    branches: 
+     - "**"
+     
+env:
+  TARGET_BRANCH: ${{ github.base_ref }}
+
+jobs:
+  run-linters:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Run npm install
+      run: npm install --save-dev
+
+    - name: Fetch target branch
+      run: git fetch -n origin ${TARGET_BRANCH}
+            
+    - name: Run phpcs linter 
+      run: |
+        FILES_DIFF=$(git diff origin/${TARGET_BRANCH} --no-commit-id --name-only -r -- '*.php')
+        if [ -n "$FILES_DIFF" ]; then
+          ./bin/phpcs.phar $FILES_DIFF --standard=phpcs-ruleset.xml
+        fi
```

How about making this more adherent to the [documentation](https://compucorp.atlassian.net/wiki/spaces/SD/pages/1933083193/Linters+workflows) and replacing the if with something like this?:

```suggestion
        [ -n "$FILES_DIFF" ] && ./bin/phpcs.phar $FILES_DIFF --standard=phpcs-ruleset.xml
```

---

So it turns out this actually doesn't work. Github runs this with the `pipefail` option, so `[ -n "$FILES_DIFF" ]` will return 1 when there are not files, and the script will stop there, marking it step as failed. [Reference](https://superuser.com/questions/1169664/bash-if-on-single-line#comment1933718_1169669)